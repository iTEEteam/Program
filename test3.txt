loadmap testmap3.txt
make human 1-0
buy tower 2-4
buy gem 2-4 range
update 1
haze on
cut on
update 2
exit
