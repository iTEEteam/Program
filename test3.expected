map testmap3.txt loaded
Human0 created
Tower0 created
update
Human0 moves
Tower0 shoots Human0
Human0 damage 20
haze is turned on
cut is turned on
update
Human0 moves
update
Human0 moves
Tower0 shoots Human0
Human0 damage 0
Human0 cut in half
Human1 created
VEGE
